integration/walletextension/wallet_extension_test.go

```diff
@@ -427,6 +429,25 @@ func TestCannotSubscribeOverHTTP(t *testing.T) {
 	}
 }
 
+func TestSuccessGasEstimate(t *testing.T) {
```

Comment:
@otherview Can we have a reverse test, that you cannot estimate gas without submitting a viewing key?

Reply:
The additional tests would test how the WE handles this (it should reject it with some meaningful error, rather than blowing up or whatever). It comes back to what we test via the new `enclave_test.go` vs here.

Tbh, I think I prefer having the tests here rather than in `enclave_test.go`, since it tests more (it also tests errors passing back to the host and through the WE). Think it maybe subjective.

@BedrockSquirrel Thoughts?